Bio/PDB/StructureBuilder.py

```diff
@@ -196,15 +195,15 @@ def init_atom(self, name, coord, b_factor, occupancy, altloc, fullname,
                                   % (duplicate_fullname, fullname,
                                      self.line_counter),
                                   PDBConstructionWarning)
-        atom = self.atom = Atom(name, coord, b_factor, occupancy, altloc,
-                                fullname, serial_number, element)
+        self.atom = Atom(name, coord, b_factor, occupancy, altloc,
+                         fullname, serial_number, element)
```

Comment:
Sorry, yes - it was already exposed as ``.atom``. My mistake.

The old version just used the local variable ``atom`` as an alias, which would be very slightly faster than ``self.atom``.